core-bundle/src/Image/Studio/ImageResult.php

```diff
@@ -142,6 +145,38 @@ public function getFilePath($absolute = false): string
         return $absolute ? $path : Path::makeRelative($path, $this->projectDir);
     }
 
+    /**
+     * Synchronously process images if they are deferred.
+     *
+     * This will make sure that the target files physically exist instead of
+     * being generated by the Contao\CoreBundle\Controller\ImagesController
+     * on first access.
+     */
+    public function createIfDeferred(): void
+    {
+        $resizer = $this->resizer();
+
+        if (!$resizer instanceof DeferredResizerInterface) {
+            throw new \RuntimeException('The "contao.image.resizer" service does not support deferred resizing.');
+        }
+
+        $picture = $this->getPicture();
+
+        $deferredImages = array_filter(
+            array_column(
+                array_merge([$picture->getImg()], $picture->getSources()),
+                'src'
+            ),
+            static function ($image): bool {
+                return $image instanceof DeferredImageInterface;
+            }
+        );
```

Comment:
I think the images from `srcset` are missing here?

Reply:
:scream: Indeed. Fixed in 71ffe3fc3a9ae59369f3f7dc08e128d7c2ff1e59.